test/run-php-linter.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
Can you update Travis to use these scripts instead of duplicating it?

(You'd have to make sure it exits with a non-zero status for any of the lint runs so that Travis will fail..)